testing/kuttl/e2e/pgbackrest-init/00-assert.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: init-pgbackrest
+status:
+  pgbackrest:
+    repoHost:
+      apiVersion: apps/v1
+      kind: StatefulSet
+      ready: true
+    repos:
+# Assert that the status has the two repos, with only the first having the `replicaCreateBackupComplete` field
+    - bound: true
+      name: repo1
+      replicaCreateBackupComplete: true
+      stanzaCreated: true
+    - bound: true
+      name: repo2
+      stanzaCreated: true
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    postgres-operator.crunchydata.com/cluster: init-pgbackrest
+    postgres-operator.crunchydata.com/pgbackrest-backup: replica-create
+    postgres-operator.crunchydata.com/pgbackrest-repo: repo1
+status:
+  succeeded: 1
+---
+# Assert the existence of two PVCs
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    postgres-operator.crunchydata.com/cluster: init-pgbackrest
+    postgres-operator.crunchydata.com/data: pgbackrest
+    postgres-operator.crunchydata.com/pgbackrest: ""
+    postgres-operator.crunchydata.com/pgbackrest-repo: repo1
+    postgres-operator.crunchydata.com/pgbackrest-volume: ""
+  name: init-pgbackrest-repo1
```

Comment:
🤔 The kuttl docs say it looks this object up by name. Does that mean it then asserts these labels exist/match?

Reply:
Yeah, afaik, if the name is given, it will retrieve that object and then match everything else given; if no name is given, it will list all objects and try to match each against everything else given.

Are you thinking checking those labels is overkill?

Reply:
I'm going to be revisiting the KUTTL tests soon, so I'll flag something like this for cleanup in my notes.